test/flake.nix

```diff
@@ -46,6 +46,15 @@
             # docs.
             foo = self.callCabal2nix "foo" (inputs.haskell-multi-nix + /foo) { };
           };
+          # WIP test
+          packageSettings = {
+            bar.input = { self, super, ... }: {
```

Comment:
I'm forced to add `{ self, super, ...}` here, in `bar.input`, rather than at the toplevel where it is more suitable to have. ie., `packageSettings = { self, super, ... }:` - however I have no idea how you would implement a deferredModule whose staticModule has a **dynamic** list of top-level options (ie., attr keys like foo, aeson, etc.). Is this even possible?